iOS Example Frame SPM/iOS Example Frame Integration Tests/CardTypeTokenCreationTests.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  iOS_Example_Frame_Integration_Tests.swift
+//  iOS Example Frame Integration Tests
+//
+//  Created by Okhan Okbay on 17/07/2023.
+//  Copyright © 2023 Checkout. All rights reserved.
+//
+
+import XCTest
+import Frames
+
+final class iOS_Example_Frame_Integration_Tests: XCTestCase {
+
+    func testTokenCreation() throws {
+        // VISA
+        verifyCardTokenCreation(cardNumber: "4485040371536584", securityCode: "123")
```

Comment:
Should we have at least 1 of each valid Visa cards ( 13 & 16 digits ) ?

Reply:
we discussed this internally and decided to not do that. I can send the link to that message that you added a thumbs up as well

Reply:
will add variations

Reply:
added